src/setup_runs/cmaq/read_config_cmaq.py

```diff
@@ -37,10 +39,10 @@ class CMAQConfig:
     run: str
     # TODO: Clarify what is meant by *short* - longer!
     """name of the simulation, appears in some filenames (keep this *short* - longer)"""
-    startDate: str = field(converter=process_date_string)
+    startDate: datetime.datetime = field(converter=process_date_string)
```

Comment:
Thanks for adjusting it. I had overlooked that one.  I also just noticed that the handling of the date is not consistent with ConfigWRF.